source/draw/FloatImage.ooc

```diff
@@ -0,0 +1,26 @@
+import math, math
+use ooc-math
+FloatImage : class {
```

Comment:
Will we ever have another kind of float image, or is this the only one? Should it be named RasterMonochromeFloat or something, instead? I'm open to discussion.